db/migrate/20250103192051_add_sizes_to_products.rb

```diff
@@ -0,0 +1,5 @@
+class AddSizesToProducts < ActiveRecord::Migration[8.0]
```

Comment:
```suggestion
class AddSizesToProducts < ActiveRecord::Migration[6.1]
```

> As for the system and feature specs, the runner is failing at bundle install and throws an argument error saying that version 8.0 is not recognized with the highest version being 7.2. This doesn't quite make sense given the fact that, locally everything works and this is a rails 8 application. The tests are being run against rails version 7.2 yet the application's migration version is 8.0.

Tests are designed to run across multiple Rails versions, `6.1`, `7.1`, and `8.0`, to verify compatibility.

If the migration version is set to `8.0`, it will fail on Rails `6.1` and `7.1`.

To resolve this, you can manually adjust the migration version to `6.1`.